bedrock/settings/base.py

```diff
@@ -703,10 +738,16 @@ def get_app_name(hostname):
     "watchman.checks.databases",
 )
 
+
+def _is_bedrock_custom_app(app_name):
+    return app_name.startswith("bedrock.")
+
+
 TEMPLATES = [
     {
         "BACKEND": "django_jinja.jinja2.Jinja2",
-        "APP_DIRS": True,
+        "APP_DIRS": False,
+        "DIRS": [f"bedrock/{name.split('.')[1]}/templates" for name in INSTALLED_APPS if _is_bedrock_custom_app(name)],
```

Comment:
This is the trick that lets us use django-jinja alongside regular Django templates. By turning off autodiscovery of apps' templates via `APP_DIRS` and instead specifying exactly what apps we want to use with django-jinja, we can let Wagtail use Django Template Language (DTL) (which it needs to work) without having to use DTL for our main Bedrock templates. Pages rendered from the CMS use jinja. It's only the templates used to render the Wagtail Admin itself that need DTL.

Also, I did try Django's own jinja2 support, but it wasn't nearly broad enough compared to django-jinja. (Mat from AMO also confirmed this and they are still on django-jinja rather than shipped-with-Django jinja2 support)

Reply:
We probably could, but I went with this way partly to be explicit that it was bedrock that needs a non-default template engine. That aside, it's a bit six-of-one-and-half-a-dozen-of-the-other.

I guess, in the future, if we drop in a third-party app (I can't think of a realistic example right now - maybe something related to l10n like django-rosetta?), I'd expect that to use DTL, so it would 'just work' with the config this way around, but that's not really a big justification

Reply:
Okay, that makes sense since bedrock is the one not using the default.